### DISCUSSION TITLE
Eu atualizei vários arquivo para chegar a uma melhoria do wingetty .

### DISCUSSION THREAD

**Discussion by eujoanderson** [thilojaeggi/WinGetty#62] [General]
Improvements we made to the Wingetty project, information that is much more useful than some that were not available:

![Captura de tela 2024-07-19 013332](https://github.com/user-attachments/assets/f8471e11-2b7c-4e39-9952-93afc8245f02)

**Comment by thilojaeggi** [comment]
Hey there.

Are these open-source?
I couldn't find them on your GitHub.

**Reply by eujoanderson** [reply]
I will fork your project. Then you could analyze it more calmly. I changed the winget_routes function because it caused a lot of problems when searching for the package. Either by spaces in the name or capital letters. 


Tonight I will fork your project. Rigth?

**Comment by eujoanderson** [comment]
Below is the link to the changes.
LINK: https://github.com/eujoanderson/Jgetty

I made changes to Winget_routes.py to be able to install applications by app name.
